save-backend/src/main/kotlin/org/cqfn/save/backend/service/AwesomeBenchmarksService.kt

```diff
@@ -0,0 +1,30 @@
+package org.cqfn.save.backend.service
+
+import org.cqfn.save.backend.controllers.AwesomeBenchmarksController
+import org.cqfn.save.backend.repository.AwesomeBenchmarksRepository
+import org.cqfn.save.entities.benchmarks.AwesomeBenchmarks
+import org.cqfn.save.entities.benchmarks.BenchmarkEntity
+import org.cqfn.save.entities.benchmarks.toEntity
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class AwesomeBenchmarksService {
+    @Autowired
+    private lateinit var awesomeBenchmarksRepository: AwesomeBenchmarksRepository
```

Comment:
Injection via constructor instead of `@Autowired`?

Reply:
sure, fixed, sorry